### ISSUE TITLE
Install Pillow-SIMD to docker images

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#1489]
## 🚀 Feature

Currently, we install Pillow to pytorch ignite docker images. Let's install Pillow-SIMD which could accelerate image processing if torchvision is used for that:
```bash
pip uninstall pillow
CC="cc -mavx2" pip install -U --force-reinstall pillow-simd
```

cc @fco-dv

**Comment by vfdev-5** [comment]
Reopen this issue as had to revert #1506

**Comment by ydcjeff** [comment]
@vfdev-5 Can you guide me how to proceed? I am new to installing package with C flags

**Comment by ydcjeff** [comment]
It can be closed now.

**Comment by vfdev-5** [comment]
Closed by #1509 and #1639
